ddtrace/tracer/option_test.go

```diff
@@ -249,6 +249,79 @@ func TestServiceName(t *testing.T) {
 	})
 }
 
+//DD_TAGS applicable only
+func TestTagSeparators(t *testing.T) {
+	assert := assert.New(t)
+
+	t.Run("env-tags", func(t *testing.T) {
+		for _, tag := range []struct {
+			in          string
+			expected    map[string]string
+			notExpected []string
+		}{
+			{
+				in:       "env:test,aKey:aVal bKey:bVal cKey:",
+				expected: map[string]string{"env": "test", "aKey": "aVal", "bKey": "bVal", "cKey": ""}},
```

Comment:
Can we please put the maps on multiple lines so it's easier to follow?
```suggestion
				expected: map[string]string{
					"env": "test",
					"aKey": "aVal",
					"bKey": "bVal", 
					"cKey": "",
				}
			},
```
Also it'd be nice if we had a lot of "sane" valid tests, as users would normally use them, or at least leave the most abusive ones last 🙂